lib/pincers/http/session.rb

```diff
@@ -45,15 +45,15 @@ def sync(_other)
     end
 
     def perform(_request)
-      perform_recursive _request, @redirect_limit, nil
+      perform_recursive _request, @redirect_limit
     end
 
   private
 
-    def perform_recursive(_request, _limit, _redirect)
+    def perform_recursive(_request, _limit)
```

Comment:
Assignment Branch Condition size for perform_recursive is too high. [29.55/25]<br>Method has too many lines. [21/15]